blocks/url-input/button.js

```diff
@@ -59,7 +59,7 @@ class UrlInputButton extends Component {
 							label={ __( 'Close' ) }
 							onClick={ this.toggle }
 						/>
-						<UrlInput value={ url || '' } onChange={ onChange } />
+						<UrlInput value={ url || '' } onChange={ onChange } data-test="UrlInput" />
```

Comment:
We shouldn't be affecting the output of the original component just to satisfy needs of unit testing. We could have achieved the same with `wrapper.find( UrlInput )`

Reply:
See #7981